lib/database/receiptserializer.js

```diff
@@ -11,8 +11,14 @@ ReceiptSerializer.prototype.encode = function(receipt, done) {
 
 ReceiptSerializer.prototype.decode = function(json, done) {
   var self = this;
-
-  this.database.transactions.get(json.transactionHash, function(err, tx) {
+  // Make sure we can handle mixed/upper-case transaction hashes
+  // it doesn't seem possible to record a transaction hash that isn't
+  // already lower case, as that's the way ganache generates them, however
+  // I don't think it will hurt anything to normalize here anyway.
+  // If you can figure out how to test this please feel free to add a test!
+  var txHash = json.transactionHash.toLowerCase();
```

Comment:
Is it worth the effort to preserve the case?  We talked briefly about potentially unintended consequences when forking.

Reply:
I changes the `get` part back to the way it was before.